v3/src/components/case-tile-common/attribute-header.tsx

```diff
@@ -60,6 +61,7 @@ export const AttributeHeader = observer(function AttributeHeader({
     prefix: instanceId, dataSet: data, attributeId
   }
   const { attributes, listeners, setNodeRef: setDragNodeRef } = useDraggableAttribute(draggableOptions)
+  useOutsidePointerDown({ ref: inputRef, handler: () => onCloseMenuRef.current?.() })
```

Comment:
I'm surprised to see `inputRef` used as the ref here. I would expect it to be `menuRef` or something similar.

Reply:
Good catch! Added a `menuListRef`